docs/docs-content/clusters/edge/local-ui/cluster-management/scale-cluster.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_label: "Scale a Cluster"
+title: "Scale a Cluster"
+description: "Learn how to scale up or scale down a cluster."
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["edge"]
+---
+
+Once a cluster is operational, you have the option of scaling up the cluster by adding additional nodes, or scale down
+the cluster by removing nodes from the cluster using Local UI.
+
+## Scale up a Cluster
+
+To scale up a cluster is to add additional nodes to an active cluster. You can scale up a cluster deployed on hosts
+installed in airgap mode in Local UI.
+
+### Prerequisites
+
+- You have an active cluster composed of hosts installed in airgap mode. For more information, refer to
+  [Create Local Cluster](./create-cluster.md).
+
+- The new nodes you plan to add to the cluster are linked with the nodes in the existing cluster. For more information,
+  refer to [Link Hosts](./link-hosts.md).
+
+- You have access to the leader node of the cluster. For more information about leader nodes, refer to
```

Comment:
We can make sure users understand every control-plane node is a leader, if its not already mentioned.

Reply:
Oh that's good to know. I didn't know every control plane node is a leader